app/com/lynxanalytics/biggraph/graph_api/MetaGraph.scala

```diff
@@ -55,23 +61,6 @@ sealed trait MetaGraphEntity extends Serializable {
     .sliding(2, 2).map(Integer.parseInt(_, 16).toByte).toSeq
 }
 
-case class StringStruct(name: String, contents: SortedMap[String, StringStruct] = SortedMap()) {
```

Comment:
Okay, so this was still used by a test. I've changed the test, but it's less thorough now. I think that's an okay trade-off. I would rather have a less precise test for a rarely used feature than having to maintain this complex code with reflection and everything.

It would also be possible to rewrite the test a bit more so it keeps the exact same check. That would be a bit more work though.